docs/typelibrary/iec61131-3/numerical/F_ATAN.md

```
@@ -1,25 +1,57 @@
-### F_ATAN
-
-
+# F_ATAN
```

markdownlint-cli2 (0.18.1)

Required heading structure
Expected: # Head; Actual: # F_ASIN

(MD043, required-headings)